web/util/session.ts

```diff
@@ -25,3 +32,47 @@ export const sessionStart = (
     sessionIndex > -1 ? minutesFromProgramStart(sessions, sessionIndex) : 0;
   return addMinutes(start, sessionStartOffset);
 };
+
+const getDuration = ({ session, duration, durationOverride }: ProgramSession) =>
+  durationOverride ?? duration ?? session?.duration ?? 0;
+
+/* Given all Programs, finds the Programs that contain the Session (matching on _id).
+ * For each match, returns:
+ * - the Program's associated Venue name (using the first entry in the Program's Venues array)
+ * - the Session's date (based on the Session's start time)
```

Comment:
As commented in a "footnote" above, I appreciate the function comment.

This doesn't mention `rawDate`, though.

Reply:
8fd3aca.